Ports/jakt/package.sh

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env -S bash ../.port_include.sh
+port=jakt
+version=git
+useconfigure=true
+depends=(llvm)
+commit_hash=063e9767ff80db1a1cfe1a805cc8b7e2e577d9f3
+archive_hash=0cb858291d0426e80c8378d7d5876a2a8de747467a289bb691782316c79a2f59
+files="https://github.com/serenityos/jakt/archive/$commit_hash.tar.gz jakt.tar.gz $archive_hash"
+auth_type=sha256
+workdir="jakt-$commit_hash"
+
+configure() {
+    run mkdir -p build
+    run mkdir -p build-host
+    run mkdir -p jakt-install
+    host_env
+    install_path="$(realpath "$workdir/jakt-install")"
```

Comment:
```suggestion
    install_path="$(realpath "${workdir}/jakt-install")"
```

Same goes for all the other usages of variables.

Reply:
🤔 any reason? Or just being uniform with other port scripts?

Reply:
That and (at least for me personally) readability. Variable names tend to be very hard to distinguish from other text otherwise.